test/e2e/suite/command/pull.go

```diff
@@ -200,6 +200,28 @@ var _ = Describe("ORAS beginners:", func() {
 				ExpectFailure().
 				MatchErrKeyWords("Error: ", "--allow-path-traversal").Exec()
 		})
+
+		It("should preserve permissions", func() {
```

Comment:
It'd be nice if we also test the case where `--preserve-permissions` is set to `false` and the permissions are not preserved.